rita/src/rita_common/payment_controller/mod.rs

```diff
@@ -153,7 +153,10 @@ impl PaymentController {
 
         let futures_chain = Box::new(transaction_status.then(move |outcome| {
             match outcome {
-                Ok(_tx_id) => {
+                Ok(tx_id) => {
+                    // add published txid to submission
+                    let mut pmt = pmt;
```

Comment:
This looks weird - isnt it redundant?

Reply:
the binding isn't mutable, we could rebind it as mutable at the top of the function but I did it down here instead

Reply:
fixed